src/Language/PureScript/AST/Exported.hs

```diff
@@ -144,3 +141,16 @@ isDctorExported ident (Just exps) ctor = test `any` exps
   test (TypeRef _ ident' Nothing) = ident == ident'
   test (TypeRef _ ident' (Just ctors)) = ident == ident' && ctor `elem` ctors
   test _ = False
+
```

Comment:
Just to be safe I tested compiling `purescript-css` (because the `CSS` module is probably the largest one I know of in terms of the number of declarations it exports), and there was no observable difference between the time it took on this branch vs the current master.